### ISSUE TITLE
Feature Request: Maximum File Size Limit

### ISSUE THREAD

**Issue by andymation** [hsoft/dupeguru#430]
Feature Request: 
In addition to the Preference for minimum file size to scan, it would be very useful to have a maximum file size option, as well. There are certain situations when I want to filter out small files and other situations when I want to prevent scanning large files in order to save time.

**Comment by chchia** [comment]
I think it will be good if the UI for this box can be larger

![image](https://user-images.githubusercontent.com/1291771/131202776-c77ed1f8-40d0-4242-8c34-9e61849150da.png)

**Comment by chchia** [comment]
![image](https://user-images.githubusercontent.com/1291771/131202856-fb6ff52e-821a-402a-9934-5c68cd6f8037.png)

i changed these 2 lines and make the it. sorry i dont know how to make PR correctly, i am not programmer~ 

![image](https://user-images.githubusercontent.com/1291771/131202867-5e129fa7-61a6-4bfd-a129-ad5ad822701b.png)

**Comment by arsenetar** [comment]
I agree they should be wider, on windows the dialog renders a bit differently so the `-` and `+` buttons take up less horizontal space.
